app/api/items/model.go

```diff
@@ -0,0 +1,62 @@
+package items
+
+import "github.com/France-ioi/AlgoreaBackend/app/database"
+
+type Item struct {
```

Comment:
Arf, static models are for ORMs.
Here, the return formats are somehow specific to the service, so not reusable. There is no such thing as database.Item or database.ItemString, the db will return a set of attribute we ask to return (with not all attributes for some tables, renamed attributes, computed attributes, attributes mixed from several tables)

Reply:
This is not aligned with your current structure of `app/database` package which groups those "sets of attributes" into separate entities. If you say that there is no such thing as `database.Item`, then you should get rid of that sepraration, the struct here (https://github.com/France-ioi/AlgoreaBackend/blob/85cb989c5942331dc0606bd559eb7a1cdfb6eb49/app/database/itemStore.go#L24) and create a set of predefined functions in your store for fetching eg. an item with all the attributes that are necessary to create something usable.